ros/src/fastrack/include/fastrack/planning/planner_manager.h

```diff
@@ -330,6 +336,46 @@ void PlannerManager<S>::TimerCallback(const ros::TimerEvent& e) {
   tf_broadcaster_.sendTransform(tf);
 }
 
+// Converts the goal state into a Rviz marker. 
+template<typename S>
+void PlannerManager<S>::VisualizeGoal() {
+  // Set up sphere marker.
+  visualization_msgs::Marker sphere;
+  sphere.ns = "sphere";
+  sphere.header.frame_id = this->fixed_frame_;
```

Comment:
Don't need `this->` here. You only need that when you're a templated derived class from a templated base and you're trying to use a member variable of your templated base.

Reply:
fixed